pages/groups/_groupId/_sessionId.vue

```diff
@@ -289,28 +285,27 @@ export default {
       return this.$vuetify.breakpoint.xs
     },
     globalInfos () {
-      const globalInfos = {
-        signed: 0,
-        present: 0,
-        away: 0,
-        not_informed: 0
-      }
+      return this.attendances
+        .reduce((acc, attendance) => {
+          const { isPresent, signDate } = attendance
 
-      this.users.forEach((user) => {
-        if (user.present === true) {
-          globalInfos.present++
-        } else if (user.present === false) {
-          globalInfos.away++
-        } else {
-          globalInfos.not_informed++
-        }
+          if (isPresent === true) {
+            acc.present++
+          } else if (isPresent === false) {
+            acc.away++
+          } else {
+            acc.not_informed++
+          }
```

Comment:
```suggestion
          if (isPresent === true) {
            acc.present++
          }
          
          if (isPresent === false) {
            acc.away++
          }
          
          if (isPresent !== false && isPresent !== true) {
            acc.not_informed++
          }
```

We have already discussed it, it would be better for the readability to make several filters. At worst, avoid `else` it is not readable